tests/python_raise/raise.py

```diff
@@ -2,10 +2,7 @@
 raise ValueError
 raise ValueError("error")
 
-raise NotImplementedError(
-    'The SimpleListFilter.lookups() method must be overridden to '
-    'return a list of tuples (value, verbose value).'
-)
```

Comment:
This seems like a useful test to have (a single long arg in the error). Or, at least, I don't think the new version is actually testing anything beyond what's tested above.